Project.toml

```diff
@@ -5,6 +5,7 @@ version = "1.2.0"
 
 [deps]
```

Comment:
Should OrderedCollections be here or does DataStructures still include OrderedDicts?